files/chef-marketplace-gem/lib/marketplace/reckoner/checker/automate.rb

```diff
@@ -0,0 +1,52 @@
+require "elasticsearch"
+
+class Marketplace
+  class Reckoner
+    class Checker
+      class Automate
+        attr_accessor :es
+
+        def initialize(opts = {})
+          @es = Elasticsearch::Client.new(
+            url: opts[:url] || Marketplace::Reckoner::Config["elasticsearch"]["url"],
+            transport_options: {
+              ssl: {
+                verify: false,
+              },
+            }
+          )
+        end
+
+        def current_usage
+          count = 0
+
+          # Limit our result set to 100 nodes and allow the scrolling identifier
```

Comment:
It just means that each section of the pagination is 100 nodes. I'm doing this just to handle the cases where somebodies node index gets huge and we don't want to return the entire thing as JSON to do the billing. As for choosing the number 100, it was totally arbitrary but seemed large enough to account for small installs and small enough as a chunk.